prog/download_boot_image.rb

```diff
@@ -8,7 +8,15 @@ def image_name
   end
 
   def custom_url
-    @custom_url ||= frame.fetch("custom_url")
+    @custom_url ||= frame["custom_url"]
+  end
+
+  def blob_storage_client
```

Comment:
let's not cache it. This will be useless after https://github.com/ubicloud/ubicloud/pull/1144. The way we make API calls are going to change. It's not necessarily needed for the usage of it here but let's keep it consistent in case we need to use other API calls in this file.

Reply:
I need to create another PR after MinIO SSL change. I will postpone this change to that PR.